ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -655,9 +658,11 @@ stdLowLevelRunNodeArgsIO RunNodeArgs{ rnProtocolInfo } StdRunNodeArgs{..} = do
             networkMagic
             (daDiffusionMode srnDiffusionArguments)
       , llrnNodeToNodeVersions =
-          supportedNodeToNodeVersions (Proxy @blk)
+            disableVersionsAbove NodeToNodeV_5
```

Comment:
For now, I've hardcoded the correct values for `blk ~ CardanoBlock`.

But note that this code is polymorphic in `blk`. So we should do one of two things:

  1. Let the user (ie `cardano-node`) supply the upper bound as a `Maybe`.
  2. Add a method to the same class that provides `supportedNodeToNodeVersions` etc so the method defines what is the upper bound (ie what is the latest officially supported/released version).

Reply:
The `Maybe` would be future-proof/flexible, but the instance would let `ouroboros-consensus` choose the default "stable" version per `blk` type, without `cardano-node` having to know what it is.

Though perhaps it should be `cardano-node` that owns that choice?

Reply:
I'd like `NodeToNodeV_6` to not be masked in the next release (tagged this week and released next week).

Reply:
I Slacked Duncan and confirmed that we want `ouroboros-network` to determine the "latest released" versions. So I went with the new method in my most recent force push.